File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/Java/Python/s063482767_commented.py
import heapq

def main():
    n = int(input())
    queue = []
    for _ in range(n):
        heapq.heappush(queue, int(input()))
    
    sum = 0
    for _ in range(n - 1):
        sum += heapq.heappop(queue)
    
    # 出力
    print("Yes" if heapq.heappop(queue) < sum else "No")

if __name__ == "__main__":
    main()  # End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/cot/Java/C/s063482767_commented.c
#include <stdio.h>
#include <stdlib.h>

// Function to compare two integers for qsort
int compare(const void *a, const void *b) {
    return (*(int*)a - *(int*)b);
}

int main() {
    int n;
    scanf("%d", &n);
    int *queue = (int*)malloc(n * sizeof(int));
    for(int i = 0; i < n; i++) {
        scanf("%d", &queue[i]);
    }
    qsort(queue, n, sizeof(int), compare);
    int sum = 0;
    for(int i = 0; i < n - 1; i++) {
        sum += queue[i];
    }
    printf(queue[n-1] < sum ? "Yes\n" : "No\n");
    free(queue);
    // End of Code
    return 0;
}
